### PR TITLE
process_resend functionality

### PR DESCRIPTION
**Pull request by tsiology**
Allows new users to resend verification email from the check-email page

### CODE REVIEW

**Comment by tsiology** [comment]
@nickolashe @dannypernik This PR from Tim to the develop branch adds the Resend email functionality from the check-email page without using a hidden input. 

Need to add explicit ordering on the token list before sending the email. Need to clean out the unnecessary migrations files.

File: openCurrents/views.py [hunk 1]
```diff
@@ -643,6 +644,47 @@ def event_register_live(request, eventid):
 
     return HttpResponse({'userid': userid, 'eventid': eventid}, status=201)
 
+# resend the verification email to a user who hits the Resend button on their check-email page
+def process_resend(request, user_email):
+    form = UserResendForm(request.POST)
```
Comment (nickolashe) [inline openCurrents/views.py]: @tsiology What is the reason we are still using a form here?

File: openCurrents/views.py [hunk 2]
```diff
@@ -643,6 +644,47 @@ def event_register_live(request, eventid):
...
+
+    user = User.objects.get(email=user_email)
+    token_records = Token.objects.filter(email=user_email)
```
Comment (nickolashe) [inline openCurrents/views.py]: @tsiology Another way to go about this is to use django queryset `last` clause and specify `get_latest_by` on the Token model. E.g., see how this is done on UserTimeLog model:
https://github.com/opencurrents/opencurrents/blob/master/openCurrents/models.py#L178

File: openCurrents/views.py [hunk 3]
```diff
@@ -643,6 +644,47 @@ def event_register_live(request, eventid):
...
+                },
+                {
+                    'name': 'TOKEN',
+                    'content': str(token)
+                }
+            ],
+            user_email
+        )
+    except Exception as e:
+        logger.error(
```
Comment (nickolashe) [inline openCurrents/views.py]: In case of error, do we still want to redirect to `check-email`?

Comment (dannypernik) [inline openCurrents/views.py]: @nickolashe I'd say yes. I believe status message is working there.

File: openCurrents/views.py (hunk 2 above)
Comment (tsiology) [inline openCurrents/views.py]: Thanks, I added get_latest_by to the Token model and implemented .last() in view

https://github.com/opencurrents/opencurrents/blob/tim/openCurrents/views.py#L653

File: openCurrents/views.py (hunk 3 above)
Comment (nickolashe) [inline openCurrents/views.py]: `check-email` template says the email message was sent, however

File: openCurrents/views.py (hunk 2 above)
Comment (nickolashe) [inline openCurrents/views.py]: great

File: openCurrents/views.py (hunk 3 above)
Comment (dannypernik) [inline openCurrents/views.py]: @nickolashe not following here

Comment (nickolashe) [inline openCurrents/views.py]: the template we are redirecting to says the email was sent while we are here because there was an error sending it

**Pull request merged**